src/components/Branding/styled-components.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import { color } from '../common-styled-components';
+
+export const BrandingContainer = styled.div`
+  position: fixed;
+  top: 0px;
+  width: inherit;
+  overflow-wrap: break-word;
+  text-align: center;
+  font-size: 28px;
+  letter-spacing: -2px;
+  margin-top: 8px;
+  margin-left: -2px;
```

Comment:
wtf is this negative margin for?

Reply:
'margin-left: -2px' to keep entire element centered. the 'letter-spacing: -2px' only affected the first span containing 'w'. After doing the math changed 'margin-left: -2px' to 'margin-left: -1px'.